src/browser/vdiff.js

```diff
@@ -17,8 +17,12 @@ mocha.setup({ // eslint-disable-line no-undef
 async function ScreenshotAndCompare(opts) {
 	const name = this.test.fullTitle();
 	const rect = this.elem.getBoundingClientRect();
-	const { pass, message } = await executeServerCommand('brightspace-visual-diff', { name, rect, opts });
-	if (!pass) {
-		expect.fail(message);
+	let result = await executeServerCommand('brightspace-visual-diff-compare', { name, rect, opts });
+	if (result.resizeRequired) {
+		this.test.timeout(0);
```

Comment:
```suggestion
		this.test.timeout(100000);
```
If we're worried about turning off the timeout, we could also pick a large number. We'll only get to this point after the screenshot is taken, so we don't need to worry about bad tests timing out - those will fail properly.  It'd only be if pixelmatch or the file operations never completed that we'd be stuck.

Reply:
I'm not worried about it -- if we run into these cases we should be able to handle them separately anyway.